### DISCUSSION TITLE
Is that possible to send email when uncaught exception on running docker container software occured?

### DISCUSSION THREAD

**Discussion by mateusz-glowinski** [nicolargo/glances#2314] [Q&A]
I'm running web application on micro-services instances application.
Is that possible to send email over `glance` when `500` was returned as response or detect and notify administrator that something went wrong?

**Comment by RazCrimson** [comment]
Nope, not possible as of now.

Could be made into a Ping/Status plugin to periodically poll an endpoint (not just specific to docker).
No support currently present for sending mails. 
Could be possible to implement with SMTP (assume a mail server is available).

Please open a feature request regarding this if you are interested.

**Reply by mateusz-glowinski** [reply]
Done: https://github.com/nicolargo/glances/issues/2317

Thanks for feedback
